web/index.js

```diff
@@ -213,7 +213,7 @@ function init() {
       if (!_services || !_template || !_variables) {
         bdk.findRoom(roomId)
           .then((res) => {
-            if (!_services) {
+            if (!_services || _.isEmpty(_services)) {
               if (res.body.settings) {
                 if (res.body.settings.onCallBotServices) {
```

Comment:
I know that's not part of the story, but can you simplify the if?
```
 if (res.body.settings && res.body.settings.onCallBotServices) {
     currentServices = res.body.settings.onCallBotServices;
 }
```

Same for variables and template.